src/main/java/com/google/cloud/storage/DatabaseQuery.java

```diff
@@ -0,0 +1,11 @@
+package google.cloud.storage;
+
+import data.Database;
+import java.util.ArrayList;
+
+public class DatabaseQuery{  
```

Comment:
What's the purpose of this class?

Reply:
The idea with this class is it handles general queries about runs for a specific database. I.e. all the runs that have been uploaded, their associated types, etc. While getting specific files the database instance is passed to the reader writer class. When getting information about the database entries themselves (i.e. likely through the spanner tables) the database instance is passed to the database query class.

Reply:
wanted to abstract away from the database class.